main.py

```diff
@@ -1 +1,22 @@
 #Mad Libs Generator Project
+loop = 1
+while (loop < 9):
```

Comment:
This causes the program to end after 8 loops to achieve 10 loops you would want to modify this implementation.